src/util/convex_polygon.h

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
To me this would make sense as a `class ConvexPolygon` with a single member variable `std::vector<v2f> polygon` and a bunch of methods

Reply:
Done.